packages/core/card.ts

```diff
@@ -1262,28 +1262,18 @@ function relationshipToCardId(ref: RelationshipObject): CardId | CardId[] | unde
   }
 }
 
-function mergeRules(rules: OcclusionRulesOrDefaults[]): OcclusionRulesOrDefaults {
+function mergeRules(rules: OcclusionRules[]): OcclusionRules {
   if (rules.length === 0) {
     throw new Error(`bug: this should never happen`);
   }
   if (rules.length === 1) {
     return rules[0];
   }
-  if (rules.find(i => i === 'everything')) {
-    return 'everything';
-  }
-  throw new Error(`unimplemented: mergeRules`);
-}
-
-// need to run this by Ed, this assumes that all interior cards will return
-// embedded field sets if the outer card was requested with a fieldset.
-function setInteriorCardFieldSet(rules: OcclusionRulesOrDefaults, interiorCardRules: OcclusionRulesOrDefaults): void {
```

Comment:
I think the policy is fine, but the place this is handled is now `rulesForField`.